source/simulator/parameters.cc

```diff
@@ -611,6 +611,11 @@ namespace aspect
                          "as opposed to continuous. This then requires the assembly of face terms "
                          "between cells, and weak imposition of boundary terms for the composition "
                          "field via the discontinuous Galerkin method.");
+      prm.declare_entry ("Use limiter for discontinuous solution", "false",
+                         Patterns::Bool (),
+                         "Whether to apply the limiter as a postprocess after obtain the discontinous solution "
```

Comment:
Please describe the limiter a bit more in this string. It is likely that users will read this without knowing about the method or purpose of the limiter.